src/stores/configurationStore.ts

```diff
@@ -0,0 +1,151 @@
+import { ConfigurationStateIF, KeyValueIF } from '@/interfaces'
+import { defineStore } from 'pinia'
+import Vue from 'vue'
+import axios from '@/axios-auth'
+
+export const useConfigurationStore = defineStore('configuration', {
+  state: (): ConfigurationStateIF => ({
+    configuration: null
+  }),
+
+  getters: {
+    getAuthWebUrl (state): string {
+      return state.configuration.VUE_APP_AUTH_WEB_URL
+    },
+
+    getMyBusinessRegistryUrl (): string {
+      return this.getAuthWebUrl + 'business'
```

Comment:
Did you test this somehow? I'm not sure if "this" works here.

Reply:
Taken from Pinia Documentation (https://pinia.vuejs.org/core-concepts/getters.html):

![pinia ref](https://github.com/bcgov/business-filings-ui/assets/122301442/31c51fcc-d41a-4719-b21f-11097b80cf2e)

Getters can call each other with `this`.

Reply:
I never doubted you 😁 

Nice reference and good job 👍